### DISCUSSION TITLE
Using functor aka EASYBUTTON_FUNCTIONAL_SUPPORT

### DISCUSSION THREAD

**Discussion by Typhoor** [evert-arias/EasyButton#64] [Q&A]
Do you have a roking exapmly for a callback ?
Do I need to use bind ?

Especially I want to callback a protected/private member method inside a object/instance who has the EasyButon instance and set's the callback and has a "this*" ...

With Ticker I can do:
  ticker.attach_ms(TICKER_DUR,+[](<class*> cc){cc->tick();}, this);

**Comment by evert-arias** [comment]
Hi @Typhoor, Please refer to the library [documentation](https://easybtn.earias.me/docs/introduction). If you find any bugs feel free to [create an issue](https://github.com/evert-arias/EasyButton/issues).
